### ISSUE TITLE
CC007 Incorrectly Flags IsNot conditions

### ISSUE THREAD

**Issue by valentine195** [apigee/apigeelint#422]
I believe this is due to the grammar checking against **Is** first, which causes a failure due to an unexpected character:

```json
{
        "source": "request.verb IsNot \"GET\"",
        "line": 327,
        "column": 17,
        "message": "Condition expression is invalid. Position 16, SyntaxError: Expected [ \\t\\n] but \"N\" found.",
        "ruleId": "CC007",
        "severity": 2,
        "nodeType": "Condition"
}
```

**Comment by ssvaidyanathan** [comment]
Ack. Until we push a fix, can you use `!=` until then?

**Comment by valentine195** [comment]
@ssvaidyanathan I just reversed the equality/inequality grammar locally, no worries.

**Comment by ssvaidyanathan** [comment]
fixed and released in v2.48.0
